src/EnergyPlus/Vectorize.hh

```diff
@@ -0,0 +1,34 @@
+#ifndef EnergyPlus_vectorize_hh_INCLUDED
+#define EnergyPlus_vectorize_hh_INCLUDED
+
+// EnergyPlus Vectorization Support
+
+// ObjexxFCL Headers
+#include <ObjexxFCL/vectorize.hh> // ASSUME_ALIGNED and RESTRICT macros
+
+// C++ Headers
+#include <cassert>
+#include <cstdint>
+
+// EnergyPlus headers
+#include "EnergyPlus.hh"
+
+namespace EnergyPlus {
+
+	int const VEC_LENGTH = VEC_ALIGN / sizeof(Real64);
+
+#define ROUND_TO_VEC_LENGTH(I) (((I) + ((VEC_LENGTH) - 1)) & ~((VEC_LENGTH) - 1))
+
+
+#define EXPLICIT_VECTORIZATION
+
+	// EXPLICIT_VECTORIZATION assumes that vectors are aligned,
+	// which is not a valid assumption on 32-bit builds
+#if defined(_WIN32) && !defined(_WIN64)
```

Comment:
Since I'm ready to PR the ObjexxFCL 4.1 release that improves on alignment/vectorization support (and other performance areas) it might make sense to rebase this branch on it.

I learned some things about getting row-aligned 2D arrays to vectorize that could help here. I could also do some profiling and vectorization work on a couple of other promising loops using the new capabilities if that is of interest -- I'm itching to give the new Intel Vectorization Advisor a spin.

I'm planning to post the ObjexxFCL PR as soon as the MArray_Elim is merged -- it may make sense to get those out of the way before we reopen the floodgates for general purpose commits.